service/resource/endpoint/spec.go

```diff
@@ -1,7 +1,7 @@
 package endpoint
 
 type Endpoint struct {
-	IP               string
+	IP               []string
```

Comment:
Nitpick: I think it should be named `IPs` now.

Reply:
fixed